internal/backend/basesvc/hello.go

```diff
@@ -17,26 +17,39 @@ var hello string
 
 var helloTemplate = template.Must(template.New("hello").Parse(hello))
 
-func sayHello(opts RunOptions, addr string) {
+func sayHello(opts RunOptions, addr, temporalFrontendAddr, temporalUIAddr string) {
```

Comment:
can we rename this to a more reasonable name?
I thought sayHello is a test util or an hello world

Reply:
what name do you suggest?

Reply:
ShowBanner, ShowServerConfig, DisplayConfig, PrintConfig, StartupMessage ...

Reply:
MOTD? (My age is showing...) :)

Reply:
Ha, went for `banner`.

Reply:
(it does not change daily :P )